scripts/script.js

```diff
@@ -0,0 +1,64 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const addBtn = document.getElementById('add-btn');
+const books = document.getElementById('books');
+
+let booksArr = [];
+
+function removeItemAt(arr, index) {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make of use array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` * instead of `push`, `pop`, `splice` and so on. (see code example below:
## EXAMPLE
`books = books.filter((book) => book.isbn !== isbn);`